test/e2e_node/jenkins/ubuntu-14.04-nvidia-install.sh

```diff
@@ -23,7 +23,7 @@ if ! dpkg-query -W cuda; then
   dpkg -i ./cuda-repo-ubuntu1404_8.0.61-1_amd64.deb
   apt-get update
   apt-get install cuda -y
-  apt-get install linux-headers-$(uname -r) -y
+  apt-get install linux-headers-"$(uname -r)" -y
```

Comment:
why change:
```
In ./test/e2e_node/jenkins/ubuntu-14.04-nvidia-install.sh line 26:
  apt-get install linux-headers-$(uname -r) -y
                                ^---------^ SC2046: Quote this to prevent word splitting.
```

Reply:
we usually quote the whole string:
```suggestion
  apt-get install "linux-headers-$(uname -r)" -y
```

Reply:
modified